css/aboutpage.css

```diff
@@ -0,0 +1,150 @@
+:root {
+  --darkColor: #272a31;
+  --brightColor: #ec5242;
+  --grayColor: #d3d3d3;
+  --whiteColor: #f7f4f4;
+  --whiteWhite: #fff;
+  --mainFont: "Lato", sans-serif;
+  --altFont1: "Carrois Gothic SC", sans-serif;
+  --altFont2: "Fredoka One", cursive;
+  --altFont3: "Hanken Grotesk", sans-serif;
+}
+
+.main-section {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.intro-title {
+  text-align: center;
+  max-width: 800px;
+}
+
+.about-statement {
+  text-align: center;
+  background-color: var(--whiteWhite);
+  border: 1px solid var(--grayColor);
+  color: rgb(34, 34, 34);
+  font-size: 1.08rem;
+  line-height: 1.7rem;
+  margin: 20px auto;
+  padding: 10px;
+  max-width: 800px;
+}
+
+.contact {
```

Comment:
Recommended to use generic classes that can be reused, to avoid repeating properties across multiple ones